### PR TITLE
rework rbac for grafana and prometheus

### PR DESCRIPTION
**Pull request by mjudeikis**
Clean rbac to prevent access

### CODE REVIEW

**Comment by mjudeikis** [comment]
Still checking where we can cut even more

**Comment by jim-minter** [comment]
/lgtm

thanks!

File: sre/prometheus-deployment/ClusterRoleBinding.authorization.openshift.io/prometheus-auth-delegator.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: prometheus-auth-delegator
+roleRef:
+  name: system:auth-delegator
+subjects:
+- kind: ServiceAccount
+  name: prometheus
+  namespace: prometheus
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply sre/prometheus-deployment/ClusterRoleBinding.authorization.openshift.io/prometheus-auth-delegator.yaml]: its little bit different deployment, so we dont want to confuse them. This is more like our own Prom deployment.

**Comment by kargakis** [comment]
Makes sense, thanks

**Pull request merged**